services/user-svc/internal/user/user.go

```diff
@@ -96,13 +53,14 @@ func HandleUserRegisteredEvent(ctx context.Context, evt *daprcmn.TopicEvent) (re
 		return true, err
 	}
 
-	db := hwgorm.GetDB(ctx)
+	var user *models.User
+	if existingUser, _ := userRepository.GetUserById(userID); existingUser == nil {
+		avatarUrl := fmt.Sprintf("%s%d", BoringAvatarURL, userID)
```

Comment:
Please hash the ID and move this into a separate repository function like generate `generateDefaultAvatarUrl(...)`